mantle/kola/tests/ignition/luks.go

```diff
@@ -176,6 +188,67 @@ func runTest(c cluster.TestCluster, tpm2 bool, threshold int, killTangAfterFirst
 	ut.LUKSSanityTest(c, tangd, m, tpm2, killTangAfterFirstBoot, rootPart)
 }
 
+func runCexTest(c cluster.TestCluster) {
+	var err error
+	var m platform.Machine
+
+	// To prevent the test to fail the whole run on s390x machine that does not have Cex Device
+	cex_uuid := os.Getenv("KOLA_CEX_UUID")
+	if cex_uuid == "" {
+		c.Skip("No CEX device found in KOLA_CEX_UUID env var")
+	}
+
+	ignition := conf.Ignition(fmt.Sprintf(`{
+		"ignition": {
+			"version": "3.5.0"
+		},
+		"kernelArguments": {
+			"shouldExist": {
+				"rd.luks.key=/etc/luks/cex.key"
+			}
+		},
+		"storage": {
+			"luks": [
+				{
+					"name": "root",
+					"device": "/dev/disk/by-label/root",
+					"cex": {
+						"enabled": %v,
+					},
+					"label": "root",
+					"wipeVolume": true
+				}
+			],
+			"filesystems": [
+				{
+					"device": "/dev/mapper/root",
+					"format": "xfs",
+					"wipeFilesystem": true,
+					"label": "root"
+				}
+			]
+		}
+	}`, true))
```

Comment:
Minor: there's no point in making this an `Sprintf` if the formatting arguments are not actually variable.